lib/app.js

```diff
@@ -25,7 +26,7 @@ app.use((req, res, next) => {
   next()
 })
 
-app.set('port', process.env.PORT || 3000)
+app.set('port', process.env.PORT || config.port)
```

Comment:
What prevent us to skip process.env.PORT ?

Reply:
retro-compatibility + the fact that `PORT` is a well known way to change port of a node app?
But I am not against removing it and only supporting the new way.

Reply:
So, with the rc configuration, the env variable would be `access_watch_port=80` not `PORT=80`.

Then, as discussed we keep `process.env.PORT`